.changeset/smart-baboons-shave.md

```diff
@@ -0,0 +1,10 @@
+---
+'@keystonejs/field-content': major
```

Comment:
This can't be here given the package no longer exists
```suggestion
```

Reply:
I think perhaps what we should do is to keep the original package, perform a `major` bump, and have that final package simply throw an error which points the dev towards the new package.

Once that final package has been published we can remove the package from the repo.

Thoughts?

Reply:
Yep, that sounds good.